services/src/main/java/org/keycloak/services/resources/admin/RoleMapperResource.java

```diff
@@ -234,6 +237,10 @@ public Stream<RoleRepresentation> getAvailableRealmRoleMappings() {
     public void addRealmRoleMappings(@Parameter(description = "Roles to add") List<RoleRepresentation> roles) {
         managePermission.require();
 
+        if (roleMapper instanceof GroupModel groupModel) {
```

Comment:
You won't need those checks at all if you are already checking this at https://github.com/keycloak/keycloak/blob/main/services/src/main/java/org/keycloak/services/resources/admin/GroupsResource.java#L128.

Reply:
It does not make sense to get role mappings for groups associated to orgs, does it?

Reply:
It's about consistency. We are blocking only update and delete operations and allowing everything else. It would be weird if some tabs on a group page would show 403 forbidden. Why then we show such a group in /groups page when you are not allowed to click on it?

Reply:
If you have this one level above you won't have tabs even shown. 

And why allow edit a group if you can not save it? Or why editing a group if they don't make sense outside the organization boundaries ....